examples/undocumented/libshogun/streaming_vwfeatures.cpp

```diff
@@ -55,12 +58,10 @@ int main()
 		SG_SPRINT("For example %d, predicted label is %f.\n", i, test_labels->get_label(i));
 
 	SG_UNREF(test_features);
-	SG_UNREF(test_file);
-	SG_UNREF(train_features);
-	SG_UNREF(train_file);
```

Comment:
@karlnapf 
Note that `CStreamingVwFeatures` will automatically free `train_file`. (in this example)

However, `CStreamingStringFeatures` will `not` automatically free `train_file`. (ref:  shogun/examples/undocumented/libshogun/streaming_stringfeatures.cpp)
We should make them work consistently.